day_6/exercises/burrito.js

```diff
@@ -22,6 +22,6 @@ var dinner = new Burrito("Beans", "Rice", ["cheese", "salsa", "guacamole"]);
 
 // Call the methods here
```

Comment:
@BrigetteDoelp note that the directions are asking you to add 3 methods to the burrito class above. What would those three methods look like? How would they update the data of a burrito instance? Please add them above and call them below.